res/mock_evm/src/metadata.rs

```diff
@@ -0,0 +1,80 @@
+use crate::*;
+use aurora_workspace_types::input::{FungibleTokenMetadata, StorageBalance, IsUsedProofCallArgs};
+
+#[near_bindgen]
+impl MockEvmContract {
+    pub fn get_version(&self) -> String {
+        "v1".to_string()
+    }
+
+    pub fn ft_total_supply(&self) -> u128 {
+        0u128
```

Comment:
In this case the compiler will understand that the type is `u128` because the hint in returning type.